app/src/main/java/org/desarrolladorslp/teckersapp/ui/messages/MessageFragment.kt

```diff
@@ -0,0 +1,31 @@
+package org.desarrolladorslp.teckersapp.ui.messages
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModelProviders
+import org.desarrolladorslp.teckersapp.R
+
+class MessageFragment : Fragment() {
+
+    private lateinit var messageViewModel: MessageViewModel
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        messageViewModel =
+            ViewModelProviders.of(this).get(MessageViewModel::class.java)
+        val root = inflater.inflate(R.layout.fragment_messages, container, false)
+        val textView: TextView = root.findViewById(R.id.text_messages)
```

Comment:
`text_messages` does not exist in this fragment's layout.

Reply:
yeah I already deleted the part of the message